### DISCUSSION TITLE
MTB FAHRRAD Navigation

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#498] [Q&A] [answered]
Der Riesengroße Vorteil von Cruiser ist die offline Planung/ Navigation. Jetzt habe ich Versuche angestellt Cruiser für die Fahrrad Navigation zu verwenden was durch die unterschiedlichen Profile sehr gut funktioniert
![Screenshot_20240417_222953_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/d7af2007-b643-4d7b-a039-ad0077eb067d)
. 
Allerdings kann man die kleinen Wege/ Radrouten bei der Planung auf der Karte schlecht sehen. Gibt es da eine Möglichkeit damit diese besser zu sehen sind ?

**Comment by devemux86** [comment]
@1maxi2 

Can you post a cycle route example to see the road types in the route?
So we can improve the map themes.

Have you tried a different map theme with menu "Map | Map theme"?
For example "Default" or "Osmarender".

You can also increase the line width in "Settings | Map | Appearance".

---

Können Sie ein Beispiel für eine Fahrradroute posten, um die Straßentypen auf der Route zu sehen?
So können wir die Kartenthemen verbessern.

Haben Sie ein anderes Kartenthema über das Menü "Karte | Kartendesign" ausprobiert?
Zum Beispiel "Standard" oder "Osmarender".

Sie können auch die Linienbreite unter "Einstellungen | Karte | Aussehen" erhöhen.

**Reply by WolfgangZ42** [reply, marked as answer]
I recommend the map theme "Motorider" - here compared with "Biker":
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b712750d-8c40-4adb-83a8-a191f61a7222"/>   <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/a16e7065-239d-410d-b315-e7741bfd76c3"/>
And, as Emux mentioned, you may further improve it by using a higher line width as 100% as in my screenshots.

-----------------------------
Ich empfehle das Kartenthema "Motorider" - hier im Vergleich zu "Biker": siehe oben.
Und, wie Emux erwähnt hat, kann man es noch verbessern, indem man eine höhere Linienbreite als 100% wie in meinen Screenshots verwendet.

**Reply by 1maxi2** [reply]
Mit " Motorider" und 120% Linienbreite funktioniert es sehr gut. Danke für den Tip !
Werde jetzt Cruiser vermehrt beim Fahrrad fahren einsetzen da zum Unterschied zu komoot auch eine offline umplanung möglich ist 😁
Allerdings machen mir die ständigen Änderungen/ Neuerungen in der app etwas zu schaffen so das ich bis auf weiteres bei der 3.0.42 bleiben werde da die für mich ! Perfekt funktioniert.
